source/standards/operating-systems.html.md.erb

```diff
@@ -26,9 +26,9 @@ Standard releases only receive security updates for 9 months after their initial
 
 ## Hardening
 
-You must apply hardening to your operating system. For example, you should disable root SSH access and deny SSH access using a password.
+You must apply hardening to your operating system. The NCSC [Cyber Assessment Framework] has a specific objective for this (B4.b). 
```

Comment:
Lower down the page we also refer to the NCSC, but we expand the acronym and then define it:

https://github.com/alphagov/gds-way/blob/88be4a67e8609b0d1056c7a6a75da2515d1adae0/source/standards/operating-systems.html.md.erb#L37

Maybe we should do that in this new line, and change the existing line to just use the acronym.